.github/workflows/molecule.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
         - edpm_nftables
         - edpm_nodes_validation
         - edpm_ovn
+        - edpm_ovs
```

Comment:
if you add it here, then 1) make default scenario run in podman; and 2) remove the ovs job from zuul config.

Reply:
yes, working on that